app/controllers/sessions_controller.rb

```diff
@@ -18,11 +24,13 @@ def create
 
   def destroy
     log_out if logged_in?
+
     redirect_to root_url
   end
 
   def add_product(product)
```

Comment:
Why do you have an add_products method in the SessionsController class when in the routes this action is mapped to products#add_to_cart?

Reply:
Whoops, forgot about this😅